### ISSUE TITLE
Carousel: on Render if a new ebay-carousel-item is added, new item is not accessible , the right arrow doesnt work. 

### ISSUE THREAD

**Issue by bhoomisatasiya** [eBay/ebayui-core#211]
# Bug Report

eBayUI Version: 0.5.0-0

## Description
on Render if a new ebay-carousel-item is added, new item is not accessible , the right arrow doesnt work. It seem on render the total width is not getting calculated to accommodate  new carousel item. 

## Screenshots
First time load:  
![screen shot 2018-06-12 at 12 06 07 pm](https://user-images.githubusercontent.com/11545472/41311411-0c4449e8-6e39-11e8-93f8-0becc3c17c15.png)

Re rendered Carousel with new item
![screen shot 2018-06-12 at 12 06 17 pm](https://user-images.githubusercontent.com/11545472/41311410-0bf11660-6e39-11e8-9608-c953eb04e2e2.png)

**Comment by DylanPiercey** [comment]
This was fixed in 0.5.0-2 (confirmed by @bhoomisatasiya).
